src/components/TotalMarketcapWidget/TotalMarketcapGQL.js

```diff
@@ -1,8 +1,13 @@
 import gql from 'graphql-tag'
 
 export const totalMarketcapGQL = gql`
```

Comment:
@haritonasty make refactoring in this PR also for the old way of names.
https://github.com/santiment/san-style-guide#naming-rule-for-graphql-query